examples/flutter_view/lib/main.dart

```diff
@@ -0,0 +1,85 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+void main() {
+  runApp(new FlutterView());
+}
+
+class FlutterView extends StatelessWidget {
+
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
```

Comment:
I'll leave it for now, as discussed yesterday, and change it later if/when we decide on another approach.